include/ebpf_nethooks.h

```diff
@@ -175,6 +176,20 @@ EBPF_HELPER(uint64_t, bpf_sock_addr_get_current_pid_tgid, (bpf_sock_addr_t * ctx
     ((bpf_sock_addr_get_current_pid_tgid_t)BPF_FUNC_sock_addr_get_current_pid_tgid)
 #endif
 
+/**
+ * @brief Set a context for consumption by a proxy application (sock_addr specific only).
+ *
+ * @param[in] ctx Pointer to bpf_sock_addr_t context.
+ * @param[in] data Pointer to data to store.
+ * @param[in] data_size The size of the data to store.
+ *
+ */
+EBPF_HELPER(void, bpf_sock_addr_set_redirect_context, (bpf_sock_addr_t * ctx, char* data, uint32_t data_size));
```

Comment:
I don't see this in Linux's helper function list, does this exist on Linux?
If not, what does Linux do for this?